### ISSUE TITLE
PHP Fatal Error when uploading some FLAC files

### ISSUE THREAD

**Issue by laurentdinclaux** [owncloud/music#545]
```
[error] 5030#5030: *9802 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Cannot 'break' 2 levels in /srv/www/nextcloud/apps/music/3rdparty/getID3/getid3/module.audio.ogg.php on line 511" while reading response header from upstream, client: xx.xx.xx.xx, server: _, request: "PUT /nextcloud/remote.php/webdav/Music/Monty%20Alexander/Concrete%20Jungle%20-%20The%20Music%20of%20Bob%20Marley/10.%20Trench%20Town.flac-chunking-2622979690-3-2 HTTP/1.0", upstream: "fastcgi://unix:/var/run/php/php7.0-fpm.sock:", host: "myhost.com"
2016/10/19 20:05:12 [error] 5030#5030: *9802 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Error: Access to undeclared static property: OC\Files\Filesystem::$normalizedPathCache in /srv/www/nextcloud/lib/private/Files/Filesystem.php:773
Stack trace:
#0 /srv/www/nextcloud/lib/private/Files/View.php(2034): OC\Files\Filesystem::normalizePath('/laurent/files/...')
#1 /srv/www/nextcloud/lib/private/Files/View.php(1152): OC\Files\View->unlockFile('/laurent/files/...', 1)
#2 [internal function]: OC\Files\View->OC\Files\{closure}()
#3 /srv/www/nextcloud/apps/files_external/3rdparty/icewind/streams/src/CallbackWrapper.php(109): call_user_func(Object(Closure))
#4 [internal function]: Icewind\Streams\CallbackWrapper->stream_close()
#5 /srv/www/nextcloud/lib/private/Files/Stream/OC.php(123): fclose(Resource id #144)
#6 [internal function]: OC\Files\Stream\OC->stream_close()
#7 {main}
  thrown in /srv/www/nextcloud/lib/private/Files/Filesystem.php on line 773" while reading upstream, client: xx.xx.xx.xx, server: _, request: "PUT /nextcloud/remote.php/webdav/Music/Monty%20Alexander/Concrete%20Jungle%20-%20The%20Music%20of%20Bob%20Marley/10.%20Trench%20Town.flac-chunking-2622979690-3-2 HTTP/1.0", upstream: "fastcgi://unix:/var/run/php/php7.0-fpm.sock:", host: "myhost.com"
```

**Comment by paulijar** [comment]
This seems to be the getID3 error fixed here: https://github.com/JamesHeinrich/getID3/pull/21. The fix is first included in getID3 version 1.9.9, but the version shipped with the Music app is 1.9.8. Obviously, we should update our getID3 version.

**Comment by paulijar** [comment]
I believe this is also a duplicate for #538
